src/maheshSantoki/arrayExamples/NonPrimeNumber.java

```diff
@@ -0,0 +1,51 @@
+/*2) Write a method to return sum of all non prime numbers 
+				and should be divisible by 5 & 7 from given array.
+				input : {11,35,17,105,44}
+				output : 140
+				Hint : (35+105 = 28)	*/
+package maheshSantoki.arrayExamples;
+
+public class NonPrimeNumber {
+	boolean flag = true;
+
+	boolean divisible(int num) {
```

Comment:
give method name clear like **divisible5And7**, it's oky if method name contains numeric & bit lengthy